massgov/mayflower-patternlab
# DP 7767 binder page navigation

## Description

Implements a table of contents overlay Organism Pattern which uses the Table of Contents Hierarchy Organism, adds an optional 'context' section to the Page Flipper Organism, adds an internal binder page Page Pattern which extends the common Binder Page content type.



## Related Issue / Ticket

https://jira.state.ma.us/browse/DP-7767

## Steps to Test


1. Look at the invision design documents as outlined here: https://jira.state.ma.us/browse/DP-7268 (see last comment)
2. Review Page Flipper With Context organism variation. Context is optional and should not affect existing Page Flippers.
3. Review Table of Contents Overlay organism. 
4. Review Internal Binder Page page. 

## Screenshots
<img width="1279" alt="screen shot 2018-02-28 at 8 57 30 am" src="https://user-images.githubusercontent.com/1397914/36791797-b7a57b92-1c66-11e8-9b01-300af121f215.png">
<img width="1277" alt="screen shot 2018-02-28 at 8 57 47 am" src="https://user-images.githubusercontent.com/1397914/36791796-b7988e78-1c66-11e8-94e4-40a035e96077.png">
<img width="1279" alt="screen shot 2018-02-28 at 8 58 02 am" src="https://user-images.githubusercontent.com/1397914/36791795-b788545e-1c66-11e8-9d8b-aad89a07020d.png">



## Additional Notes:

I'd say this is a minor change, as it introduces brand new functionality, and adds a variation in a backwards-compatible way - i.e no breaking changes. 

#### Impacted Areas in Application

* additional variation `@organism/page-flipper~with-context`
* new `@orgamism/table-of-contents-overlay`
* new `@page/internal-binder-page`

#### @TODO




#### Today I learned...
